cinema/serializers.py

```diff
@@ -2,0 +1,84 @@
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, Movie, MovieSession, CinemaHall
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.CharField(read_only=True)
+
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
```

Comment:
Consider adding type annotations to the `Meta` class methods to improve code readability and maintainability. This is a requirement according to the task checklist.